config/locales/es.yml

```diff
@@ -1475,6 +1475,14 @@ mfa.second_method_warning.link: Agregue un segundo método de autenticación.
 mfa.second_method_warning.text: Deberá eliminar su cuenta y comenzar de nuevo si pierde su único método de autenticación.
 mfa.skip: Omitir por ahora
 mfa.webauthn_platform_message: Agregue otro método de autenticación en caso de que pierda el acceso al primero.
+multiple_accounts_detected.created_at_html: '<strong> Created: </strong> %{timestamp_html}'
+multiple_accounts_detected.heading: We believe you may have another account
+multiple_accounts_detected.intro: We have found another %{app_name} account that we believe may be yours. Do you recognize the following accounts as yours?
+multiple_accounts_detected.last_sign_in_at_html: '<strong> Last login: </strong> %{timestamp_html}'
+multiple_accounts_detected.yes_many: Yes, these are my accounts
+multiple_accounts_detected.yes_single: Yes, this is my account
+mutliple_accounts_detected.no_recognize_single: I do not recognize this account
```

Comment:
Curious, why "don't" and "do not" here and the next one?

Reply:
IO think thats down to a design choice. I think don't vs do not doesnt matter to me open to both.